### PR TITLE
Add Workflow Stable proto API

### PR DESCRIPTION
**Pull request by antontroshin**
# Description

Add Workflow Stable proto API
Add deprecation notice on Workflow Alpha1 API

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [ ] Code compiles correctly
- [ ] Created/updated tests
- [ ] Unit tests passing
- [ ] End-to-end tests passing
- [ ] Extended the documentation / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_
- [ ] Specification has been updated / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_
- [ ] Provided sample for the feature / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by elena-kolevska — CHANGES_REQUESTED** [review]
You'll also need to add the methods in `pkg/api/universal/workflow.go`, update the endpoints and ideally, update e2e tests. You can refer to [this PR](https://github.com/dapr/dapr/pull/6911) for guidance on things that are needed when upgrading an API.

**Comment by antontroshin** [comment]
> You'll also need to add the methods in `pkg/api/universal/workflow.go`, update the endpoints and ideally, update e2e tests. You can refer to [this PR](https://github.com/dapr/dapr/pull/6911) for guidance, I think it will be helpful.

It is indeed very helpful, thanks a lot!

**Comment by elena-kolevska** [comment]
Heya @antontroshin  pls ping when this is ready for review 🙏

**Comment by antontroshin** [comment]
@elena-kolevska Sorry missed the comment, this is ready for review. TY.

**Review by mikeee — COMMENTED** [review]
Will these endpoints be retained solely used for HTTP invocations - as the dapr workflow grpc endpoints are being retired

E.g.  https://github.com/dapr/go-sdk/pull/640

**Review by elena-kolevska — APPROVED** [review]
LGTM, but @mikeee's question is a good one. If we're deprecating the "main client" workflows in all sdks, will we need to add the stable endpoints?

File: dapr/proto/runtime/v1/dapr.proto [hunk 1]
```diff
@@ -191,6 +205,28 @@ service Dapr {
...
+
+  // Pauses a running workflow instance
+  rpc PauseWorkflow (PauseWorkflowRequest) returns (google.protobuf.Empty) {}
+
+  // Resumes a paused workflow instance
+  rpc ResumeWorkflow (ResumeWorkflowRequest) returns (google.protobuf.Empty) {}
+
+  // Raise an event to a running workflow instance
+  rpc RaiseEventWorkflow (RaiseEventWorkflowRequest) returns (google.protobuf.Empty) {}
+
```
Comment (mikeee) [inline dapr/proto/runtime/v1/dapr.proto:229]: As mentioned yesterday I think these methods should be removed

**Review by mikeee — CHANGES_REQUESTED** [review]

**Comment by antontroshin** [comment]
Closing. Workflow APIs are being deprecated (https://github.com/dapr/dapr/pull/8299) and the Workflow client will be used instead.

**Pull request closed without merging**